code/src/components/NewTodo.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import todo from 'reducers/todo'
+import uniqid from 'uniqid'
+import styled from 'styled-components'
+import { Section } from './styling/Wrappers'
+
+const NewTodo = () => {
+  const dispatch = useDispatch()
+  const [newTodo, setNewTodo] = useState('')
+
+  const onFormSubmit = (event) => {
+    event.preventDefault()
+    const postNewTodo = {
+      id: uniqid(),
+      name: newTodo,
+      isCompleted: false
+    }
+    dispatch(todo.actions.addItem(postNewTodo))
```

Comment:
Your naming here, addItem(postNewTodo) is very helpful in reading and understanding the code.